mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -119,9 +119,11 @@ def random_flip_data_3d(self, input_dict, direction='horizontal'):
         if 'centers2d' in input_dict:
             assert self.sync_2d is True and direction == 'horizontal', \
                 'Only support sync_2d=True and horizontal flip with images'
-            w = input_dict['img_shape'][1]
+            w = input_dict['ori_shape'][1]
             input_dict['centers2d'][..., 0] = \
                 w - input_dict['centers2d'][..., 0]
+            input_dict['cam_intrinsic'][0][2] = \
+                w - input_dict['cam_intrinsic'][0][2]
```

Comment:
Can we add a comment to explain this? This is a bit hard to understand at first glance for beginners.

Reply:
Done.